src/main/java/com/minecolonies/coremod/colony/buildings/BuildingHome.java

```diff
@@ -24,10 +31,36 @@
  */
 public class BuildingHome extends AbstractBuildingHut
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2160)